web/service/build.gradle

```diff
@@ -49,7 +49,7 @@ dependencies {
 	implementation 'com.google.cloud:spring-cloud-gcp-starter-storage'
   implementation 'org.projectlombok:lombok:1.18.22'
   implementation 'org.projectlombok:lombok:1.18.22'
-  implementation 'io.sentry:sentry-spring:1.7.30'
+  implementation 'io.sentry:sentry-spring-boot-starter:6.17.0'
```

Comment:
question: do we need this change as part of the PR? It seems unrelated to the scope.